packages/component-utils/lib/core/util/test.ts

```diff
@@ -0,0 +1,17 @@
+declare const Cypress: any
+
+/**
+ * Get a key for a specific test, by name
+ */
+export function key (): string {
+  // Split out for unit testing purposes
+  if (Cypress && Cypress.mocha) {
+    const runner = Cypress.mocha.getRunner()
+
+    if (runner && runner.test) {
+      return (runner.test || '') + Math.random()
+    }
+  }
+
+  return ''
+}
```

Comment:
what is this `key` for? i see it becomes part of the `ComponentTestInstance`, but i don't see anywhere where it's actually read after that, in this package or in the current `cypress-(vue|react)-unit-test` repos

Reply:
It's used by the consumer (framework-specific API). React uses it when mounting the component.